shinysdr/i/network/export_http.py

```diff
@@ -71,19 +70,20 @@ def __init__(self, block, wcommon, deleteSelf):
                 if cell.type().is_reference():
                     self._blockCells[key] = cell
                 else:
-                    self.putChild(key, ValueCellResource(cell, self.__wcommon))
+                    self.putChild(key.encode(), ValueCellResource(cell, self.__wcommon))
```

Comment:
This will use the default encoding when on Python 2, but should always be UTF-8.

Reply:
Done